openshift_scalability/PrometheusLoader.md

```diff
@@ -0,0 +1,15 @@
+# Intro
+this tools used to fire prometheus queries against prometheus instance in OpenShift
+stack context.
+
+It's also helps to simulate Grafana workloads, on staging cluster.
```

Comment:
Editorial:  "It also helps  to simulate Grafana workloads on a staging cluster."